src/wfs/src/test/java/org/geoserver/wfs/v2_0/GetCapabilitiesTest.java

```diff
@@ -685,6 +685,16 @@ public void testInternationalContent() throws Exception {
         assertXpathEvaluatesTo("titolo italiano", fifteenLayer + "/wfs:Title", doc);
         assertXpathEvaluatesTo("abstract italiano", fifteenLayer + "/wfs:Abstract", doc);
         assertXpathEvaluatesTo("parola chiave", fifteenLayer + "/ows:Keywords/ows:Keyword", doc);
+
```

Comment:
Same as above. Also it would be better to have this code in a separate test method.

Reply:
[[https://github.com/geoserver/geoserver/pull/5179/commits/ce2b333c58071213a39f7f7aa04ee99948834d1b#diff-18da30d7affb394d894f5a2bb175784e5fcfac3b58c6cfece7755dcbfed7beb9R176-R197](url) added a new test and removed the previous modifications.](url) added a new test and removed the previous modifications.